package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "blot",
+  "version": "1.0.0",
```

Comment:
Let's use semantic versioning for our packages. 
[semver.org](https://semver.org)
We can start with 0.1.0 for this PR.